styles/variables-base.less

```diff
@@ -159,8 +159,6 @@
 
 // DateTimePicker
 // ---------------------------------------
-@agate-datetimepicker-margin: 6px 0;
-@agate-datetimepicker-border-radius: 0;
 @agate-datetimepicker-transform: none;
 @agate-datetimepicker-label-transform: none;
 @agate-datetimepicker-cell-transform: none;
```

Comment:
What are these for?

Reply:
Those were left from the last version of DateTimePicker. I wanted to remove them, but I thought that styling was out of scope. I will remove them, though, as I changed the component's structure.